### DISCUSSION TITLE
Can't get image rows() & columns()

### DISCUSSION THREAD

**Discussion by Tarifut** [ImageMagick/ImageMagick#2995] [Help]
test.jpg is a 634x458 jpg

Image magicImage;
magicImage.read( "test.jpg" );
int height = magicImage.rows();
int width= magicImage.columns();

cout << width << "-" << height << endl;  

Displayed return is 458x8

**Comment by dlemstra** [comment]
Which library are you using? Nevermind.. just noticed it was the cpp api.

Which version of ImageMagick are you using?

**Comment by Tarifut** [comment]
I use magick++ last version (imagemagick 7.0.10-49)

**Comment by dlemstra** [comment]
Can you also share your test image?

**Reply by Tarifut** [reply]
https://img.lignes-formations.com/wp-content/uploads/sites/45/2019/05/formation-reportage-photo-niveau-1.jpg

**Comment by dlemstra** [comment]
I am unable to reproduce your issue. Did you manage to resolve this?
